spec/features/follow.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+feature "follow another user" do
+  given!(:action) {Fabricate(:category)}
+  given!(:movie1) {Fabricate(:video, title: "Scandal", category: action)}
+  given!(:movie2) {Fabricate(:video, title: "Lost", category: action)}
+  given!(:movie3) {Fabricate(:video, title: "Shiva", category: action)}
+
+  given!(:user1) {Fabricate(:user)}
+  given!(:user2) {Fabricate(:user)}
+
+  given!(:review1) {Fabricate(:review, user: user2, video: movie1)}
+
+  background do
+    feature_sign_in(user1)
+    find("a[href='/videos/#{movie1.slug}']").click
+    find("a[href='/users/#{user2.slug}']").click
```

Comment:
`feature_sign_in(user1)` < --- this is abstracted to `feature_sign_in` method.

`find("a[href='/videos/#{movie1.slug}']").click` this is a raw code kinda. You should make an abstraction on the same level - `click_video_link(movie1)` and the implementation of the method is your homework. :)